src/Organic/AdminSettings.php

```diff
@@ -190,6 +190,8 @@ public function showSettings() {
         $ad_slots_prefill_enabled = $this->organic->getOption( 'organic::ad_slots_prefill_enabled' );
         $log_to_sentry = $this->organic->getOption( 'organic::log_to_sentry', true );
 
+        $ads_enabled = $this->organic->getOption( 'organic::ads_enabled' );
```

Comment:
This means that after the plugin upgrade, the Ads will become disabled because the `organic::ads_enabled` field in DB will be empty

If the `organic::ads_enabled` is empty, we should pre-populate it based on behavior of previous versions of the plugin: if plugin/integration is enabled and configured, then set it to `true`

Reply:
Good call, thank you!